CruGlobal/staff-conf-onsite
# Active admin views for child and student medical

To test you will need to have rails running , reddis running and sidekiq running.

The hybrid import data I am running is the following csv content located here:
https://gist.github.com/kharperballistiq/a6976c7b734deaeed3939ad03bda81b7#file-cru-merged-sample-csv-medical-history-csv

To test a working example navigate to :
http://localhost:3000/families


![image](https://user-images.githubusercontent.com/26168983/51139377-14482700-181a-11e9-83ac-78b560e52790.png)
-=-=-=-=-=-=-=-=-=-=-=-=


Click on "**Import Spreadsheet**" in the top right to access the import screen
![image](https://user-images.githubusercontent.com/26168983/51139444-51141e00-181a-11e9-98bb-2d884f1caebc.png)
-=-=-=-=-=-=-=-=-=-=-=-=


Then choose the sample CSV file listed above and click submit.

After the import is complete, you can click on People>Children

Now you should see two children, one is under 5 years old, the other is over 5 years old.  If you View each one you will see the under 5 medical history for the young kid and the over 5 student info for the child over 5 years old.

You should be able te edit the information in the medical history for each child by clicking Edit.
